src/Haskell/Ide/Engine/Plugin/Package.hs

```diff
@@ -338,11 +338,19 @@ codeActionProvider plId docId _ context = do
 -- | Extract a module name from an error message.
 extractModuleName :: T.Text -> Maybe Package
 extractModuleName msg
-  | T.isPrefixOf "Could not find module " msg = Just $ T.tail $ T.init nameAndQuotes
-  | T.isPrefixOf "Could not load module " msg = Just $ T.tail $ T.init nameAndQuotes
+  | T.isPrefixOf "Could not find module " msg = Just $ extractTerm line
+  | T.isPrefixOf "Could not load module " msg = Just $ extractTerm line
   | otherwise = Nothing
   where line = head $ T.lines msg
-        nameAndQuotes = T.dropWhileEnd (/= '’') $ T.dropWhile (/= '‘') line
+
+extractTerm :: T.Text -> T.Text
```

Comment:
Can we somehow reduce this code duplication?

Reply:
You mean moving `extractTerm`to a common module? Yeah, that is my intention but i am not sure where to place it (`askell.Ide.Engine.PluginUtils`?)

Reply:
I would rather go for HieExtras.